spec/features/audit/report/allocation_spec.rb

```diff
@@ -1,31 +1,34 @@
 RSpec.feature "Content Allocation", type: :feature do
-  let!(:my_organisation) do
-    create(
-      :organisation,
-    )
-  end
-
-  let!(:me) do
-    create(
-      :user,
-      organisation: my_organisation,
-    )
+  scenario "Filter allocated content" do
+    given_i_am_auditing_a_content_item
+    then_i_see_all_content_items
+    given_that_i_filter_by_attributes_that_do_not_apply_to_any_content_items
+    then_i_see_no_content_items
   end
 
-  scenario "Filter allocated content" do
-    create(
+  def given_i_am_auditing_a_content_item
+    organisation = create(:organisation)
+    user = create(:user, organisation: organisation)
+    content_item = create(
       :content_item,
       title: "Do Androids Dream of Electric Sheep",
-      allocated_to: me,
-      primary_publishing_organisation: my_organisation,
+      allocated_to: user,
+      primary_publishing_organisation: organisation,
     )
+    @audit_report_page = ContentAuditTool.new.audit_report_page
+    @audit_report_page.load(content_id: content_item.content_id)
```

Comment:
This is a user action - the user visiting the page. It belongs in its own `when` step.